mne/io/fiff/raw.py

```diff
@@ -156,18 +156,21 @@ def _read_raw_file(self, fname, allow_maxshield, preload, compensation,
             raw_node = dir_tree_find(meas, FIFF.FIFFB_RAW_DATA)
             if len(raw_node) == 0:
                 raw_node = dir_tree_find(meas, FIFF.FIFFB_CONTINUOUS_DATA)
-                if (len(raw_node) == 0) and allow_maxshield:
+                if (len(raw_node) == 0):
                     raw_node = dir_tree_find(meas, FIFF.FIFFB_SMSH_RAW_DATA)
-                    msg = ('This file contains raw Internal Active Shielding '
-                           'data. It may be distorted. Elekta recommends it be'
-                           ' run through MaxFilter to produce reliable '
-                           'results. Consider closing the file and running '
-                           'MaxFilter on the data.')
-                    info['maxshield'] = True
-                    warnings.warn(msg)
-
-            if len(raw_node) == 0:
-                raise ValueError('No raw data in %s' % fname)
+                    if (len(raw_node) == 0):
+                        raise ValueError('No raw data in %s' % fname)
+                    elif allow_maxshield:
+                        msg = ('This file contains raw Internal Active '
+                               'Shielding data. It may be distorted. Elekta '
+                               'recommends it be run through MaxFilter to '
+                               'produce reliable results. Consider closing '
+                               'the file and running MaxFilter on the data.')
+                        info['maxshield'] = True
+                        warnings.warn(msg)
+                    else:
+                        raise ValueError('IAS detected, '
```

Comment:
I think we should spell out IAS, I doubt most people are familiar with the term (even some who use it)